cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,23 @@
+/// <reference types="cypress" />
+
+describe('', () => {
+  it('', () => {
```

Comment:
You should name your test. '' is not an informative test name ('' in describe and it)